src/components/Button/index.tsx

```diff
@@ -19,6 +19,7 @@ interface Props {
   disabled?: boolean;
   dataCy?: string;
   glyph?: React.ReactElement;
+  size?: Size;
```

Comment:
I know you didnt start this but instead of recreating each type can we instead inherit the prop types from the `LeafyGreenButton`

Reply:
Button prop interfaces aren't exported from leafygreen so I think this is the best we can do

Reply:
You should be able to use this to extract the prop types from the leafygreen `type ButtonProps = React.ComponentProps<typeof LeafyGreenButton>`